mldsa/src/sign.h

```diff
@@ -686,4 +687,25 @@ __contract__(
   ensures(return_value <= MLD_DOMAIN_SEPARATION_MAX_BYTES)
 );
 
+/*************************************************
+ * Name:        pk_from_sk
```

Comment:
Should this be renamed to `crypto_sign_pk_from_sk` to align with the other APIs?

Reply:
done!